external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -29,45 +30,36 @@ if(${ENABLE_PCMSolver})
                        -DCMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER}
                        -DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_BINDIR=${CMAKE_INSTALL_BINDIR}
+                       -DPYMOD_INSTALL_LIBDIR=${PYMOD_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_DATADIR=${CMAKE_INSTALL_DATADIR}
                        -DCMAKE_INSTALL_INCLUDEDIR=${CMAKE_INSTALL_INCLUDEDIR}
+                       -DENABLE_psi4=ON
                        -DSTATIC_LIBRARY_ONLY=${_a_only}
+                       -DSHARED_LIBRARY_ONLY=${_so_only}
 #                       -DENABLE_OPENMP=???
```

Comment:
I currently have no OpenMP parallelization in PCMSolver, nor have I enabled the one that comes "for free" with Eigen. For that option, you can pass the value set when configuring Psi4, it won't hurt (that I know of)